support/builder/config.sh

```diff
@@ -38,6 +38,7 @@ provider = "$OAUTH_PROVIDER"
 token_url = "$OAUTH_TOKEN_URL"
 userinfo_url = "$OAUTH_USERINFO_URL"
 redirect_url = "$OAUTH_REDIRECT_URL"
+signup_url = "$OAUTH_SIGNUP_URL"
```

Comment:
I believe this is a no-op?

Reply:
I don't think so, the builder-api-proxy uses this variable to populate the habitat.conf.js file?

Reply:
Yes, but this line is in the builder-api config, not the builder-api config

Reply:
See https://github.com/habitat-sh/builder/blob/master/components/oauth-client/src/config.rs#L32 for the builder-api oauth config

Reply:
Ah sorry yes from the perspective of the app this should be a no op, this isn't used inside the actual oauth config and it doesn't need to be, its just for propagating the value to the proxy via the front end webapp. We have to have this here so that the link default gets overriden in the front end, nothing more.

Reply:
I definitely don't understand why that would be the case - the builder-api-proxy config should be sufficient for propagating the link to the front end...

Reply:
I 100% thought this was the builder-api-proxy config: you are correct it looks like I added this to the api config as well. I'll get that cleaned up!